src/components/landing/CommunityListingHeader.js

```diff
@@ -0,0 +1,98 @@
+import CustomizableAvatar from "../common/customavatar/CustomizableAvatar";
+import Link from "next/link";
+import Image from "next/image";
+
+const CommunityHeader = ({ listing }) => {
+  const photo = "/community dashoard  img 1.jpg";
+  const Manager_Avatar = "/avarar 1.png";
+  const AVATAR_WIDTH = 50;
+  const AVATAR_HEIGHT = 50;
+  const SmallAvatar_WIDTH = 20;
+  const SmallAvatar_HEIGHT = 20;
+  const LOGO_WIDTH = 20;
+  const LOGO_HEIGHT = 20;
+  return (
+    <header className="">
+      <div className=" bg-primary-lite p-5 md:flex lg:flex items-center justify-between px-10">
+        <article className="flex items-center pb-4">
+          <CustomizableAvatar
+            src={photo}
+            width={AVATAR_WIDTH}
+            height={AVATAR_HEIGHT}
+            avatarImage={"rounded-full"}
+            avatarCustomClass={"mx-2"}
+          />
+          <h1 className="text-grey-dark text-xl font-medium capitalize">
+            {listing?.community_name}
+          </h1>
+        </article>
+
+        <div className="flex items-center px-4">
+          <div className="flex items-center bg-white rounded-lg py-2 px-4 text-justify">
+            <CustomizableAvatar
+              src={Manager_Avatar}
+              width={SmallAvatar_WIDTH}
+              height={SmallAvatar_HEIGHT}
+              className={"rounded-full"}
+            />
+            <p className="text-primary text-sm text-center pl-1 capitalize">
+              {listing?.community_manager}
+            </p>
+          </div>
+
+          <Link
+            href={
+              listing?.social_links?.twitter !== undefined
+                ? listing?.social_links?.twitter
+                : ""
+            }
+            className={"px-4"}
+          >
+            <Image
+              src="/twitter.svg"
+              alt="Twitter Logo"
+              width={LOGO_WIDTH}
+              height={LOGO_HEIGHT}
+            />
+          </Link>
+          <Link href={"#"}>
+            <Image
```

Comment:
I believe we agreed on the social icons being `twitter` and `linkedin` (not `facebook`).

These links are 2 ways that users can get in touch with the Community Manager. I doubt that Facebook is top of those ways. 😉  Linkedin and Twitter are more professional.

Reply:
I will amend accordingly